apps/src/p5lab/P5Lab.js

```diff
@@ -212,26 +212,33 @@ export default class P5Lab {
     this.isBlockly = this.studioApp_.isUsingBlockly();
 
     this.skin = config.skin;
-    if (this.isBlockly) {
-      const mediaUrl = `/blockly/media/spritelab/${config.level
-        .instructionsIcon || 'avatar'}.png`;
-      this.skin.smallStaticAvatar = mediaUrl;
-      this.skin.staticAvatar = mediaUrl;
-      this.skin.winAvatar = mediaUrl;
-      this.skin.failureAvatar = mediaUrl;
+    let mediaUrl;
+    switch (this.getLabType()) {
+      case P5LabType.GAMELAB:
+        mediaUrl = null;
```

Comment:
i know this is already merged, but we should offload this logic to the subclass (same as we're doing with `this.getLabType()`). then, this P5Lab logic could look like:

```javascript
let mediaUrl = this.mediaUrl();
if (mediaUrl) {
  mediaUrl += `${config.level.instructionsIcon || 'avatar'}.png`;
}
```

we would just have to standardize on whether or not `this.mediaUrl()` has a trailing slash